app/embedder.py

```diff
@@ -127,11 +127,11 @@ def _buildEmbedding(self, seq: str) -> np.array:
 
 
 @embed_bp.route('/embed', methods=['POST'])
-def make_embed():
+def embed():
     error = None
```

Comment:
I like the #request.get_json: {... comment from classify() to document the API.

Reply:
done